wellcomeml/ml/frequency_vectorizer.py

```diff
@@ -83,23 +83,9 @@ def spacy_lemmatizer(self, X, remove_stopwords_and_punct=True):
 
         """
 
-        try:
-            nlp = spacy.load(
-                "en_core_web_sm", disable=["ner", "tagger", "parser", "textcat"]
-            )
-        except IOError:
-            from wellcomeml.__main__ import download
-
-            download("models")
-            # pkg_resources need to be reloaded to pick up the newly installed models
-            import pkg_resources
-            import imp
-
-            imp.reload(pkg_resources)
-            nlp = spacy.load(
-                "en_core_web_sm", disable=["ner", "parser", "textcat"]  # Spacy 3.0 needs the tagger to lemmatise
-            )
-
+        nlp = spacy.blank("en")
+        nlp.add_pipe("lemmatizer", config={"mode": "lookup"})
```

Comment:
Nice find. I think this needs `spacy-lookup-data` to be install (which is probably installed automatically with `spacy[lookup]`.

Reply:
just did a little experiment with a blank virtualenv and it seems like as long as `spacy[lookups]` is installed then in the script `import spacy` is enough for this to work. Or am I confusing something?

Reply:
can you try what happens if `en_core_web_sm` model is not installed? I think an IOError is thrown still so you should keep the exception handling that @aCampello had

Reply:
seems to work without it being installed:
```
>>> import spacy
>>> nlp=spacy.load('en_core_web_sm')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/gallaghe/Code/nutrition-labels/build/virtualenv/lib/python3.7/site-packages/spacy/__init__.py", line 47, in load
    return util.load_model(name, disable=disable, exclude=exclude, config=config)
  File "/Users/gallaghe/Code/nutrition-labels/build/virtualenv/lib/python3.7/site-packages/spacy/util.py", line 329, in load_model
    raise IOError(Errors.E050.format(name=name))
OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a Python package or a valid path to a data directory.
>>> nlp = spacy.blank("en")
>>> nlp.add_pipe("lemmatizer", config={"mode": "lookup"})
<spacy.lang.en.lemmatizer.EnglishLemmatizer object at 0x15ad7ca00>
>>> nlp.initialize()
<thinc.optimizers.Optimizer object at 0x15aba8050>
>>> X = ['the cats sat on the sitting room floors enjoying the sun']
>>> [[token.lemma_.lower() for token in doc] for doc in nlp.pipe(X)]
[['the', 'cat', 'sit', 'on', 'the', 'sit', 'room', 'floor', 'enjoy', 'the', 'sun']]
```

Reply:
ok cool, just realised that you initialise a blank model, good 👍 thanks for checking

Reply:
Yeah, that's actually a big advantage. You just need to install the lookups, not the entire model! Much simpler code too.